### DISCUSSION TITLE
Folder selection improvement ?

### DISCUSSION THREAD

**Discussion by tompalmer** [theophanemayaud/video-simili-duplicate-cleaner#125] [Ideas]
Hi, 

As of now, if I want to select a folder, I can only do 1 by 1. Could a system with checkboxes that bypasses macos be used ? Would save a lot of time

**Comment by theophanemayaud** [comment]
Hey, sorry for being so late but thanks a lot for the suggestions! This is a great and common suggestion. I’ll work on it soon™️ in #123
